### ISSUE TITLE
Provide a default db.json in README.md so that no errors comes up on setting janitor locally.

### ISSUE THREAD

**Issue by arshdkhn1** [JanitorTechnology/janitor#183]
Running `node app` and other commands on fresh install give this error -
```
[2017-10-29T16:04:16.981Z] [fail] letsencrypt { Error: getaddrinfo EAI_AGAIN acme-v01.api.letsencrypt.org:443
    at Object.exports._errnoException (util.js:1020:11)
    at errnoException (dns.js:33:15)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:76:26)
  code: 'EAI_AGAIN',
  errno: 'EAI_AGAIN',
  syscall: 'getaddrinfo',
  hostname: 'acme-v01.api.letsencrypt.org',
  host: 'acme-v01.api.letsencrypt.org',
  port: 443 }
```
There are no instructions given on README.md to solve this.

**Comment by arshdkhn1** [comment]
db.json should have below configurations by default - 
```js
  "security": {
    "forceHttp": true,
    "forceInsecure": true
  }
```
and other configurations for sign in.

**Comment by jankeromnes** [comment]
Indeed, this is a problem. I really like the idea of mentioning this in the readme. Also you're suggesting great defaults for db.json, in fact they're already in use when you develop Janitor on Janitor, see the default db.json in dockerfiles https://github.com/JanitorTechnology/dockerfiles/blob/master/janitor/db.json

**Comment by jankeromnes** [comment]
This was fixed in 05bc614ae74412865666dd9755faec448c539704

Thanks @KiLlOrBe!
